instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+"""
+This library contains all android_binary and android_instrumentation_test targets to each
+test suite in instrumentation module as a oppia_instrumentation_test wrapper.
+Note that:
+ - All the oppia_instrumentation_test targets are named similar to the respective class name
+ of the test suite.
+"""
+
+exports_files(["src/javatest/AndroidManifest.xml"])
+
+# Used for end-to-end tests
+android_binary(
+    name = "oppia_test",
```

Comment:
Added it here considering it would be easier for the 2.3 pr
which might avoid using the explort_files for src/java/AndroidManifest.xml

Reply:
Seems reasonable.

Does this mean that we can remove the exports_files declaration? That's not ideal.

Reply:
What I meant here is when we add a new AndroidManifest.xml in the in instrumenation/src/java
we would have to add export_files it when we use the oppia_test in the root BUILD.bazel

Reply:
Ah I see. Doesn't the file just need to be relative to wherever the test is declared? I.e. if we move this manifest to be next to the ExplorationPlayerTest.kt file, will it work?

Reply:
Currently this binary uses the manifest from app module I've added the binary under instrumentation such that when we create the test application we can directly access the manifest from the instrumentation/src/java/AndroidManifest.xml

Incase we use the oppia_test binary in the root we would have to add export_files for the above manifest